.github/workflows/build.yml

```diff
@@ -18,6 +18,10 @@ jobs:
         with:
           # Need to fetch 2 commits for the PR (base commit and head merge commit) so we can compute the diff
           fetch-depth: 2
+      - uses: actions/setup-python@v2
```

Comment:
what is this new file? I am not aware of this.

Reply:
the github action upgraded their python version and is conflicting with existing code in `parseq-tracevis`